src/System.Data.SqlClient/tests/stress/StressTest/Constants.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DPStressHarness
+{
+    static public class Constants
```

Comment:
Should be "public static". There are also some other instances of static being in front of things.